dev/src/db_utils.py

```diff
@@ -1,83 +1,38 @@
-from typing import Any, cast
+from typing import Any
 
-from psycopg import Cursor, sql
+from sql import Column, Table  # type: ignore
```

Comment:
Add the following section to the `setup.cfg` to not have to add `# type: ignore` everytime you import something from the `sql` package:
```
[mypy-sql]
ignore_missing_imports = true
```

Reply:
That's what I did yesterday and now tried again, but the errors stayed.